tests/test_buffers.py

```diff
@@ -119,10 +136,17 @@ def test_device_buffer(replay_buffer_cls, device):
         DictRolloutBuffer: DummyDictEnv,
         ReplayBuffer: DummyEnv,
         DictReplayBuffer: DummyDictEnv,
+        RecurrentRolloutBuffer: DummyEnv,
+        RecurrentDictRolloutBuffer: DummyDictEnv,
     }[replay_buffer_cls]
     env = make_vec_env(env)
 
-    buffer = replay_buffer_cls(100, env.observation_space, env.action_space, device=device)
+    if replay_buffer_cls == RecurrentRolloutBuffer:
+        buffer = RecurrentRolloutBuffer(
```

Comment:
Optional:
Here and below, `100` comes off as a bit of a magic number that it's hard to see why it was picked (and what it does). Passing all these in as kwargs would make it a little more readable.

If this 100 is the 'same' 100 that is used in line 153, it would be good to refactor them to use a shared constant.